betamax/serializers/proxy.py

```diff
@@ -27,6 +27,13 @@ def __init__(self, serializer, cassette_path, allow_serialization=False):
         self.cassette_path = cassette_path
 
     def _ensure_path_exists(self):
+        directory, _ = os.path.split(self.cassette_path)
+        if directory != '':
+            if not os.path.isdir(directory):
+                raise OSError(
```

Comment:
I'm not sure I like raising an `OSError`, a `RuntimeError` might be better.

Perhaps it would also be better to just *make* the directory. Since this isn't something people will likely be doing as a one-off thing (or repetitively) I would doubt we'll be creating tons of typo'd directories for people. Thoughts?